fetch.bs

```diff
@@ -1408,18 +1469,50 @@ for each associated <a for="fetch group">fetch record</a> whose
 <a>connection</a> is identified by an <b>origin</b> (an
 <a for=/>origin</a>) and <b>credentials</b> (a boolean).
 
+<h4 id=connection-reuse>HTTP/2 Connection Reuse</h4>
```

Comment:
editorial details question:  fetch.bs has not (yet?) converted over to the "modern" bikeshed markup, so would y'all really want @vanupam using that as in suggestions above, without converting the entire file ?